tools/packages/wesl-plugin/src/weslPlugin.ts

```diff
@@ -256,44 +279,42 @@ function makeGetWeslMain(
  */
 async function loadWesl(
   context: PluginContext,
-  unpluginCtx: UnpluginBuildContext,
+  unpluginCtx: UnpluginBuildContext & UnpluginContext,
 ): Promise<Record<string, string>> {
-  const { options } = context;
-  const { weslFiles } = await getWeslToml(context, unpluginCtx);
-  const { weslToml } = options;
-  const tomlDir = weslToml ? path.dirname(weslToml) : process.cwd();
-
-  const globs = weslFiles.map(g => tomlDir + "/" + g);
-  const futureFiles = globs.map(g => glob(g));
+  const {
+    toml: { weslFiles },
+    resolvedWeslRoot,
+    tomlDir,
+  } = await getWeslToml(context, unpluginCtx);
+  const futureFiles = weslFiles.map(g =>
+    glob(g, { cwd: tomlDir, absolute: true }),
+  );
   const files = (await Promise.all(futureFiles)).flat();
   // dlog({ files, weslRoot, tomlDir, globs });
-  return loadFiles(files, tomlDir);
+  return await loadFiles(files, resolvedWeslRoot);
```

Comment:
We're now returning paths relative to the wesl root, instead of paths that are relative to the toml.